README.md

```diff
@@ -4,12 +4,15 @@
 
 > Jupyter React is a set of [React.js](https://reactjs.org) components that allow a frontend developer to build data products compatible with the [Jupyter](https://jupyter.org) ecosystem. The user interface delivers executable notebooks and cells.
 
+> Jupyter React allows [Jupyter](https://jupyter.org) components to be embedded into [React.js](https://reactjs.org) applications as regular react components.
+
 <div style="background-color: rgb(230,246,230); font-size: large; text-align: center; padding: 10px; margin: 10px 0 10px 0;">
 Read the documentation on <a href="https://jupyter-react.datalayer.tech" target="_blank">https://jupyter-react.datalayer.tech</a>
 </div>
 
 The below image shows a gallery of the available React.js components ready to be used in you custom application. These open source components are used to build the online [Datalayer service](https://datalayer.app), a collaborative platform for data analysis.
```

Comment:
Can you make an explanatory diagram of what the app is doing?  from first glance this looks like another UI for jupyter lab.  Which I get is your use case,  what about the dev who is familiar with reactjs, and wants to embed a notebook component into their app (do these people exist)?

Reply:
> Can you make an explanatory diagram of what the app is doing

A diagram would look like this... will that help?

```
User
  |
React.js Application
  |
Jupyer React
  |
JupyterLab ------------------------------- Jupyter Server
```

> Which I get is your use case, what about the dev who is familiar with reactjs, and wants to embed a notebook component into their app

True

> do these people exist?

From the activity here and at other places, they do exist. The question then is "how many?"

Reply:
As an aside, I wish there was a good 5-10 minute overview of the python data ecosystem, especially highlighting pandas, matplotlib, and jupyter notebook.  Wes Mckinney used to have a great 10 minute introduction to pandas (recorded 10 or so years ago), that has now been taken down (by him presumably).

The video should just quickly show some of the things pandas/jupyter is capable of, with the takeaway being "wow this is a cool programming environment I should learn about", instead most videos I see are much slower paced because the videos try to explain how to use pandas, starting from first concepts.

This would be a great place to link to such a video.